### ISSUE TITLE
How to use vad-filter in python version of WhisperX?

### ISSUE THREAD

**Issue by kurianbenoy-sentient** [m-bain/whisperX#139]
Thanks for this awesome open source package.

I am looking for the python equivalent code for CLI command:

```
whisperx examples/sample01.wav --model large-v2 --vad_filter --align_model WAV2VEC2_ASR_LARGE_LV60K_960H
```

It looks like in python code, vad-filter argument is not supported. Is it correct? Can the team know how we can enable vad-filter by default to get better timestamp accuracy?

```
import whisperx

device = "cuda" 
audio_file = "audio.mp3"

# transcribe with original whisper
model = whisperx.load_model("medium.en", device)
result = model.transcribe(audio_file)

print(result["segments"]) # before alignment

# load alignment model and metadata
model_a, metadata = whisperx.load_align_model(language_code="en", device=device, model_name="WAV2VEC2_ASR_LARGE_LV60K_960H")

# align whisper output
result_aligned = whisperx.align(result["segments"], model_a, metadata, audio_file, device)

print(result_aligned["segments"]) # after alignment
print(result_aligned["word_segments"]) 
```

**Comment by MDOR69** [comment]
I gotchu fam

import whisperx
from pyannote.audio import Inference
import torch

#File you want to transcribe
audio_file = "path to file here"

#Load the VAD model whisperx uses
hugging_face_token = "huggingface token here"
device = "cuda"

#Create the VAD pipeline
vad_pipeline = Inference(
            "pyannote/segmentation",
            pre_aggregation_hook=lambda segmentation: segmentation,
            use_auth_token=hugging_face_token,
            device=torch.device(device),
        )

# transcribe with whisper + VAD Parallel
model = whisperx.load_model("small.en", device)
result = whisperx.transcribe_with_vad_parallel(model=model,audio=audio_file, batch_size=4, vad_pipeline=vad_pipeline,language='en',task="transcribe")

print(result["segments"]) # before alignment

# load alignment model and metadata
model_a, metadata = whisperx.load_align_model(language_code=result["language"], device=device)

# align whisper output
result_aligned = whisperx.align(result["segments"], model_a, metadata, audio_file, device)

**Comment by kurianbenoy-sentient** [comment]
Thank you @MDOR69

**Comment by zongtao975** [comment]
Thank you @MDOR69  but , I can't find “transcribe_with_vad_parallel” in asr ， Can you help me ？
